### PR TITLE
Node 12 fixes

### PR DESCRIPTION
**Pull request by DullReferenceException**
- Tweak `_refresh` logic so tests pass in node 12
- Add `.eslintrc` so IDEs can show linting errors
- Fix mismatching dependencies
- Fix reported audit vulnerabilities

### CODE REVIEW

File: lib/multi-level.js [hunk 1]
```diff
@@ -135,39 +135,38 @@ class MultiLevelCache {
...
-          throw new TypeError('shouldCache has to be a function');
-        }
-
-        // Given that we are not ignoring this value, perform an out-of-band cache update
-        if (shouldCache(cacheItem.value)) {
-          // NB: an in-band update would `await` this Promise.all block
-          Promise.all(this._caches.map(cache => cache.set(key, cacheItem))).catch(err => {
-            throw new Error(`Error caching ${key}`, err);
-          }).then(() => {
+      this._pendingRefreshes[key] = (async () => {
```
Comment (DullReferenceException) [inline lib/multi-level.js:138]: In short, the change is to make the clearing of the pending refreshes occur within the refresh operation instead of after.

Comment (msluther) [reply lib/multi-level.js:138]: I like the change, but might be more readable to pull out the async function and just make this code something like:

```js
this._pendingRefreshes[key] = this._forceRefresh(key, staleItem, updateFn, opts);
```

Comment (DullReferenceException) [reply lib/multi-level.js:138]: @msluther: love the suggestion, will do so.

Comment (DullReferenceException) [reply lib/multi-level.js]: @msluther: done.

**Pull request merged**